document steps for configuring okta provider (#807)
RoadieHQ/marketing-site: content/docs/integrations/okta/index.md

---
title: Configuring Okta Team / User Provider
publishedDate: '2022-08-02T17:00:00.0Z'
description: How to configure Roadie to collect users from Okta.

humanName: Okta
integrationType: OSS plugin
---

## Introduction

> In order to use the Okta organization plugin, Roadie needs an Okta API token, and the organization URL to communicate with Okta APIs. On this page, you'll learn how to set up this API token in your Roadie Admin panel. 

Reviewer: ```suggestion
In order to use the Okta organization plugin, Roadie needs an Okta API token, and the organization URL to communicate with Okta's APIs. On this page, you'll learn how to set up these two fields in your Roadie Admin panel. 
```

---
title: Configuring Okta Team / User Provider
publishedDate: '2022-08-02T17:00:00.0Z'
description: How to configure Roadie to collect users from Okta.

humanName: Okta
integrationType: OSS plugin
---

## Introduction

> In order to use the Okta organization entity plugin, Roadie needs an API token, and the organization url to communicate with Okta APIs.
> 
> This page describes how to create and set up the API token.

Reviewer: ```suggestion
In order to use the Okta organization plugin, Roadie needs an Okta API token, and the organization URL to communicate with Okta APIs. On this page, you'll learn how to set up this API token in your Roadie Admin panel. 
```

## Prerequisites

> You will need to [configure an Okta API token](https://developer.okta.com/docs/guides/create-an-api-token/) for use by Backstage.

Reviewer: ```suggestion
Obtain an [Okta API token](https://developer.okta.com/docs/guides/create-an-api-token/) to use with Roadie Backstage. 
```

Also, I suggest you change this from "Prerequisites" to "Step 1"

Author: done. thanks

## Steps

### Step 1: Set the Okta API token

> Visit the secrets page in the Backstage UI and set the `OKTA_TOKEN` secret.

Reviewer: ```suggestion
Set the `OKTA_TOKEN` secret in the secrete manager of your Roadie Backstage Admin panel, located in the following path:
```

```text
https://<tenant-name>.roadie.so/administration/settings/secrets
```

### Step 2: Set the Okta Organization URL

> Visit the Okta configuration page in Backstage and set the Organization URL.

Reviewer: ```suggestion
Set the Okta Organization URL in the Okta configuration page of your Roadie Backstage Admin panel, located in the following path:
```
